python/lsst/ts/Dome/mock_llc_statuses/amcs_status.py

```diff
@@ -50,24 +45,30 @@ def __init__(self, period):
         self.resolver_raw = np.zeros(_NUM_MOTORS, dtype=float)
         self.resolver_calibrated = np.zeros(_NUM_MOTORS, dtype=float)
 
-    async def determine_status(self):
+    async def determine_status(self, current_tai):
```

Comment:
Please document the new argument in the doc string.

Reply:
It is documented in the BaseMockStatus class. As far as I know, there is no way to refer to the doc string of the super class but if there is I'd like to know and I will add it to the six sub classes.